packages/abi-typegen/src/templates/contract/dts.hbs

```diff
@@ -40,6 +40,13 @@ export type {{structName}}Output{{typeAnnotations}} = { {{outputValues}} };
 {{/if}}
 {{/each}}
 
+{{#if formattedConfigurables}}
+export type {{capitalizedName}}Configurables = {
+  {{#each formattedConfigurables}}
+    {{configurableName}}: {{configurableType}};
+  {{/each}}
```

Comment:
This fixes the double-indentation on the _rendered_ template.

The rendered fixtures will need to be adjusted accordingly.

```suggestion
{{#each formattedConfigurables}}
  {{configurableName}}: {{configurableType}};
{{/each}}
```